### PR TITLE
Refactor Movie and Customer

### PR DESCRIPTION
**Pull request by cristinahica**
Removed undescores from the attribute names.

### CODE REVIEW

**Review by raaulb10 — APPROVED** [review]
respect

File: clean-code-java-master/src/main/java/videostore/horror/Movie.java [hunk 1]
```diff
@@ -3,23 +3,23 @@ public class Movie {
 	public static final int CHILDRENS = 2;
 	public static final int REGULAR = 0;
 	public static final int NEW_RELEASE = 1;
-	private String _title;
-	private Integer _prc;
+	private String title;
```
Comment (cristinahica) [inline clean-code-java-master/src/main/java/videostore/horror/Movie.java:6]: this is related to clean code

File: clean-code-java-master/src/main/java/videostore/horror/Customer.java [hunk 2]
```diff
@@ -44,17 +44,20 @@ public String statement() {
 			// add frequent renter points
 			frequentRenteerPoints++;
 			// add bonus for a two day new relaese rental
-			if ( each.getPrC() != null &&
```
Comment (TeoBradatan) [inline clean-code-java-master/src/main/java/videostore/horror/Customer.java:47]: cinstit

**Review by TeoBradatan — CHANGES_REQUESTED** [review]
bg

**Review by TeoBradatan — APPROVED** [review]
bg

**Review by rtudorr22 — APPROVED** [review]

**Pull request closed without merging**